<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fm_principal</class>
 <widget class="QMainWindow" name="fm_principal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SLL - Sistema LifeLogging - Lembrando você de tudo que você fez.</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource_files.qrc">
    <normaloff>:/icons/images/icone.png</normaloff>:/icons/images/icone.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="pushButton_acao_comum1">
        <property name="text">
         <string>Tomei água</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_acao_comum2">
        <property name="text">
         <string>Peidei</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_acao_comum3">
        <property name="text">
         <string>Fumei um cigarro</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>New Life Logging</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QComboBox" name="comboBox_eventos"/>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_inserir">
            <property name="text">
             <string>Inserir</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Search</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Inicial:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDateEdit" name="dateEdit_data_inicial"/>
          </item>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Final:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDateEdit" name="dateEdit_data_final"/>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_pesquisar"/>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_pesquisar">
            <property name="text">
             <string>Pesquisar</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Outro:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_outro">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="3" column="0">
     <widget class="QTableWidget" name="tableWidget_life_log"/>
    </item>
    <item row="4" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Versão &lt;span style=&quot; font-weight:700; color:#00007f;&quot;&gt;5.0&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_autenticacao">
        <property name="text">
         <string>pushButton</string>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_logado">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_fechar">
        <property name="text">
         <string>Fechar</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionColaboradores"/>
    <addaction name="actionEventos"/>
    <addaction name="actionSobre"/>
    <addaction name="actionSair"/>
   </widget>
   <addaction name="menuMenu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSobre">
   <property name="text">
    <string>Sobre</string>
   </property>
  </action>
  <action name="actionSair">
   <property name="text">
    <string>Sair</string>
   </property>
  </action>
  <action name="actionEventos">
   <property name="text">
    <string>Eventos</string>
   </property>
  </action>
  <action name="actionColaboradores">
   <property name="text">
    <string>Colaboradores</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>pushButton_acao_comum1</tabstop>
  <tabstop>pushButton_acao_comum2</tabstop>
  <tabstop>pushButton_acao_comum3</tabstop>
  <tabstop>comboBox_eventos</tabstop>
  <tabstop>pushButton_inserir</tabstop>
  <tabstop>tableWidget_life_log</tabstop>
  <tabstop>pushButton_autenticacao</tabstop>
  <tabstop>pushButton_fechar</tabstop>
 </tabstops>
 <resources>
  <include location="resource_files.qrc"/>
 </resources>
 <connections/>
</ui>
